src/js/pages/jobs/JobsTaskDetail.js

```diff
@@ -0,0 +1,178 @@
+import classNames from 'classnames';
+import {Confirm} from 'reactjs-components';
+
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import ChronosStore from '../../stores/ChronosStore';
+import TaskDetail from '../services/task-details/TaskDetail';
```

Comment:
Agreed, definitely wanted to do it elsewhere to prevent massive conflicts/confusion. Issue has been created in JIRA